src/core/pluginmanager.h

```diff
@@ -103,6 +89,9 @@ class PluginManager : public QObject
     Q_INVOKABLE bool isAppPluginEnabled( const QString &uuid ) const;
     Q_INVOKABLE bool isAppPluginConfigurable( const QString &uuid ) const;
 
+    bool isAppPluginLocallyAvailable( const QString &name ) const;   // we must use uuid instead of name!
+    bool isAppPluginPublicalyAvailable( const QString &name ) const; // we must use uuid instead of name!
```

Comment:
Get rid of "public" / "publicly". On the API side, we should refer to Remote / Remotely.